roles/container-engine/meta/main.yml

```diff
@@ -1,5 +1,12 @@
 ---
 dependencies:
+  - role: container-engine/kata-containers
```

Comment:
The role of kata-containers should definitely be launched before the containerd ?

Reply:
When kata-containers is enabled, containerd role will use the [configuration](https://github.com/kubernetes-sigs/kubespray/pull/6256/files/aaf181f4d6b77b04cc8088578ac45ea86b23036e#diff-72d387cef1abe0943eb10028f704b0bf) and the [executables](https://github.com/kubernetes-sigs/kubespray/pull/6256/files/aaf181f4d6b77b04cc8088578ac45ea86b23036e#diff-444d7e8480a53338aa2d82a5649554daR21) created by the kata-containers role. Anyway this containerd configuration is not used until the [RuntimeClass are created](https://github.com/kubernetes-sigs/kubespray/pull/6256/files/aaf181f4d6b77b04cc8088578ac45ea86b23036e#diff-623927c432abf196b9405aa7342880abR25) in Kubernetes, so it's safe to change the launch order.

It isn't mandatory to put kata-containers before containerd, but I see that it makes sense because it provisions files that will be used by the containerd role. Same behaviour should happens for other container runtimes in the future.